### ISSUE TITLE
"Bridge" holidays

### ISSUE THREAD

**Issue by mblasi** [rickar/cal#67]
Hello, here in Argentina, we have a rule for promote turism, which adds Friday as holiday when the real one is Thursday, and adds Monday when it is Tuesday.
Is there any way to represent it with the new v2 api?

Thanks!

**Comment by rickar** [comment]
There's no built-in way to do that. You should be able to create a custom HolidayFn that returns Friday or Tuesday after checking if the original holiday is on Thursday or Tuesday.

**Comment by mblasi** [comment]
Thank you @rickar! If I understand correctly, that HolidayFn would swap the "Thursday" with "Friday", but in my case, we need to add Friday, and finally should be two holidays: Thursday and Friday.

**Comment by rickar** [comment]
Right. Each Holiday definition can only return one day per year so you would need to create something like TruthDay and SecondTruthDay. TruthDay would return the normal holiday every year like you currently have it with CalcDayOfMonth and SecondTruthDay would use a custom CalcBridgeDay that returns the extra day if it applies or zero time if not.

**Comment by mblasi** [comment]
I see. So I should define this "second" day for each actual holiday...
